src/javascript/views/components/common/Sidebar.jsx

```diff
@@ -75,91 +75,117 @@ class Sidebar extends React.Component {
     }
 
     render() {
+        let component;
+        if (this.props.configLoading) {
+            component = (
+                <div className="zk-ui-loader"></div>
+            );
+        } else {
```

Comment:
maybe early return? it will save an indent level which makes the rest a bit harder to read.

```
if (this.props.configLoading) {
           return (
               <div className="zk-ui-loader"></div>
           );
       } 
```